### PR TITLE
simdutf 2.1.0 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by pthariensflame** [comment]
I'm not sure what to do about the audit error (it didn't happen on my system).  I just tested the suggestion, replacing both `depends_on`s with `uses_from_macos`s and the formula failed to build.  It *might* be okay to do that for only `libiconv`—the build succeeds, but the resulting `sutf` binary has less capability—but switching it for `icu4c` just doesn't work.

**Comment by pthariensflame** [comment]
Anyone able to set "help requested" or similar?

**Comment by chenrui333** [comment]
`uses_from_macoss` could be false alarm, just use `depends_on` instead.

**Review by chenrui333 — DISMISSED** [review]

File: Formula/simdutf.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/simdutf/simdutf.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => :build
+  depends_on "icu4c"
+  depends_on "libiconv"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_TESTING=ON"
```
Comment (chenrui333) [inline Formula/simdutf.rb:15]: `std_cmake_args` should come at last as default

```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DBUILD_TESTING=ON", *std_cmake_args
```

**Review by chenrui333 — DISMISSED** [review]

**Review by pthariensflame — COMMENTED** [review]
@chenrui333 This change caused a failure to build, because `std_cmake_args` sets `BUILD_TESTING` to `OFF`.

**Comment by chenrui333** [comment]
aha, that is good to know.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @pthariensflame!!

**Pull request closed without merging**